citrination_client/search/client.py

```diff
@@ -39,6 +39,10 @@ def pif_search(self, pif_system_returning_query):
         :return: :class:`PifSearchResult` object with the results of the query.
         :rtype: :class:`PifSearchResult`
         """
+
+        if sum(filter(None, [pif_system_returning_query.from_index, pif_system_returning_query.size])) >= 50000:
```

Comment:
What is the purpose of the filter here? Maybe I'm missing something, but couldn't you just say:
```
if pif_system_returning_query.from_index + pif_system_returning_query.size >= 50000
```
Also, I would define the 50000 in a constant at the top of the files. MAX_QUERY_RESULTS or something like that.

Finally, I don't actually understand why from_index is being added to size. Is it because we don't allow access past 50k entries? If so, I would probably format the error message such that that is more clear.

Reply:
I would guess that this is to handle the case where `from_index` or `size` are null or None?

Reply:
https://en.m.wikipedia.org/wiki/Zen_of_Python

> Explicit is better than implicit; If the implementation is hard to explain, it's a bad idea.

Reply:
@joannehill I assume sean is right about your intent here; maybe add a comment explaining?